TRITON/TRITON-IOC-01App/Db/TRITON_pid_lookup.db

```diff
@@ -111,6 +113,23 @@ record(ao, "$(P)LKUP:TEMP:SP:RBV")
 	field(FLNK, "$(P)LKUP:LUTON:SP")
 }
 
+#
+# This is a filthy hack but it's the only way I found to get
+# around ReadASCII's way of initializing records.
+#
+# Essentially, assume that ReadASCII will have finished initialising
+# by the time 10 seconds have elapsed, and then write to the LUTON:SP
+# record by channel access to force ReadASCII to recognise the value change.
+# 
```

Comment:
note to reviewer: I added https://github.com/ISISComputingGroup/IBEX/issues/4174 to find a better solution for this.